### DISCUSSION TITLE
Two Questions About Terms of Service

### DISCUSSION THREAD

**Discussion by SpaceOmega5000** [community/community#41703] [Other Feature Feedback, Questions, & Ideas]
Hello, in the terms of service, there is a provision stating that you own Your Content in the short and longer provision under _D. User Generated Content_, but it doesn't really specify what ownership it's talking about? Is it just Intellectual Property rights or does it go beyond that too? "G. Intellectual Property Notice" maybe makes this more clear for GitHub's content but not so much for user generation content. I'm a bit worried that such broadness of ownership is similar to how certain EULAs try to take away certain rights (certain rights _beyond_ IP rights, like physical ownership of copy) against end-users who receives their own copy and is offline. This is probably ridiculous fear, but as an open source enthusiastic, I do strictly worry about things like this.

Also do these terms and it's survival apply only on the GitHub service? Is there special contractual status over users who terminate contract when they no longer use GitHub? Or would their legal status be naturally the same as if they never used and never planned to use GitHub once they terminate contract?

I apologize if this isn't the best place to ask. I would love if anyone who made the terms of service to give out answers to my questions, but I am not sure if this is the best place and can't find a easy way the creators directly.

**Comment by LuisContrerasGlz** [comment]
The vittles in the terms of service regarding power of stoner- generated content relate to intellectual property rights. Specifically, it means that you retain power of the brand in the content that you produce and post on GitHub, but you grant GitHub a license to use, display, and distribute that content. This allows GitHub to give the service of hosting and participating your content with others.

These terms and conditions apply to your use of the GitHub service, whether you're laboriously using the service ornot.However, the terms and conditions will continue to apply to any content that you have formerly posted on the service, If you terminate your account or stop using the service.
